kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -1002,7 +1002,7 @@
       },
       allowLearnersDownloadDescription: {
         message:
-          "Allow users to explore resources they don't have and mark it for Kolibri to automatically download when it's available in their network.",
+          'Allow users to explore resources they don’t have and mark it for Kolibri to automatically download when it’s available in their network.',
```

Comment:
See my note above about consistent punctuation. I think we probably don't want to be using the `‘`. Either leave with the double quotes, or escape the `\'`.